pkg/manager/risingwave_controller_manager_impl.go

```diff
@@ -186,6 +186,9 @@ func (mgr *risingWaveControllerManagerImpl) CollectOpenKruiseRunningStatisticsAn
 			Type: buildObjectStorageType(objectStorage),
 		}
 
+		versions := strings.Split(mgr.risingwaveManager.RisingWave().Spec.Global.Image, ":")
```

Comment:
The image could be 
- an empty string, `""`
- a well-formed image, `"ghcr.io/risingwavelabs/risingwave:v0.1.16"`
- a simplified one, `"ghcr.io/risingwavelabs/risingwave"`, which is identical to the `"ghcr.io/risingwavelabs/risingwave:latest"`

Please revisit if the splitting and array indexing work well in all cases.

Reply:
In the latest commit:
1. Extract the function to `common.go` file.
2. Deal with the case you mentioned. I check the document and also deal with the case image:port:version one. For exmaple, from `ghcr.io/risingwavelabs/risingwave:10043` to  `latest`, from `ghcr.io/risingwavelabs/risingwave:10043:v0.1.16` to `v0.1.16`
3. Add unit test in `comment_test.go`

Not sure if I cover all the cases. If anything missing, feel free to tell me :)